src/test/java/BaseTest.java

```diff
@@ -1,12 +1,53 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
 
+import java.time.Duration;
+
 
 public class BaseTest {
+public static WebDriver driver = null;
 
     @BeforeSuite
     static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
+    @BeforeMethod
+    public static void launchBrowser() {
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+    }
+        @AfterMethod
+                public static void closeBrowser() {
+            driver.quit();
+        }
+            protected static void navigateToPage() {
+                String url = "https://bbb.testpro.io/";
+                driver.get(url);
+            }
+                public static void provideEmail(String email) {
+                    WebElement emailField = driver.findElement(By.cssSelector("[type=email]"));
+                    emailField.clear();
+                    emailField.sendKeys(("Needajob@gmail.com"));
+                }
+                public static void providePassword(String password) {
+                    WebElement passwordfield = driver.findElement(By.cssSelector("[type= password]"));
+                    passwordfield.clear();
+                    passwordfield.sendKeys("I'mready123!");
+                }
+                public static void clickSubmit() throws InterruptedException {
+        WebElement submitButton = driver.findElement(By.cssSelector("button[type='submit']"));
+        submitButton.click();
+        Thread.sleep(2000);
 
 }
+
+
+    }
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)